bins/src/bin/datadog-static-analyzer-server.rs

```diff
@@ -51,6 +51,25 @@ impl Fairing for CORS {
     }
 }
 
+#[rocket::get("/shutdown")]
+fn shutdown_get(server_configuration: &State<ServerConfiguration>) -> Status {
+    if server_configuration.is_shutdown_enabled {
+        Status::NoContent
```

Comment:
Why not `200` or `Ok`? After all, if we can shutdown, it's okay?

Reply:
We could have used 200, sure. But in this case, a 204 is also a success code that indicates that no content is expected from that endpoint. I mean, I don't have a strong opinion here aside than what I already stated, so if you prefer returning a 200 code, I'd be fine with it. 

I now realise that I used a 200 for the post version, which doesn't return any content either. I would unify both codes with 204, or 200 if you prefer that for some other reason.